Source/m_menu.c

```diff
@@ -2743,10 +2743,11 @@ setup_menu_t keys_settings1[] =  // Key Binding screen strings
   {"STRAFE"      ,S_INPUT     ,m_scrn,KB_X,KB_Y+9*8,{0},input_strafe},
   {"AUTORUN"     ,S_INPUT     ,m_scrn,KB_X,KB_Y+10*8,{0},input_autorun},
   {"VERTICAL MOUSE",S_INPUT   ,m_scrn,KB_X,KB_Y+11*8,{0},input_novert},
+  {"MOUSELOOK"   ,S_INPUT     ,m_scrn,KB_X,KB_Y+12*8,{0},input_mouselook},
```

Comment:
Is it "mouselook" or "mouse look"? Help!

Reply:
It's "mouse look" in Quake 1.06
![image](https://user-images.githubusercontent.com/5077629/159637425-703c5ce3-4ee4-489c-b581-c5416a048e0b.png)

Reply:
But "mouselook" in GZDoom: https://zdoom.org/wiki/Mouse_options

Reply:
😱

Reply:
I'm not sure what a UK English speaker would think, but "mouselook" feels more correct to me here on the other side of the Atlantic.

Reply:
That's enough for me, thank you! 😎